circuit_setup/scripts/run_setup.sh

```diff
@@ -58,7 +58,7 @@ if [ -f "${CIRCOM_SRC_DIR}/circomlib" ]; then
     rm -f "${CIRCOM_SRC_DIR}/circomlib"
     if [[ "$OS" == "Windows_NT" || "$(uname -o 2>/dev/null)" == "Msys" ]]; then
         echo "Creating Windows junction..."
-        cmd //c "mklink /J ${CIRCOM_SRC_DIR//\//\\}\\circomlib circuits\\circomlib"
+        cmd //c "mklink /J $(cygpath -wa "$CIRCOM_SRC_DIR"/circomlib) $(cygpath -wa "${ROOT_DIR}/circuits/circomlib")"
```

Comment:
Is cygpath always defined, even if you don't use a cygwin type system?

Reply:
fixed

Reply:
> Is cygpath always defined, even if you don't use a cygwin type system?

`cygpath` will only be used on windows/msys (gitbash) shells. If on ubuntu, the symlink will already be there and cygpath won't get called.